frontend/src/components/StoreList/StoreList.scss

```diff
@@ -23,6 +23,10 @@
   &:hover {
     color: $color-primary;
   }
+
+  &.--is-hovered {
```

Comment:
We usually don't do the `.` in front so that it takes the Block's name in front like `Block--modifier`